### PR TITLE
UI updates and bug fixes

### LINKED ISSUE #220: Sharing glb seems to be broken
Steps to repro:
- Go to https://playcanvas.com/viewer
- load glb from url https://playcanvas.github.io/static/assets/models/playcanvas-cube.glb
- click on share button on the bottom, and notice that the link does not contain the glb url at all: `https://playcanvas.com/viewer/?load=[object Object]`
- QR code does not work either, as it uses the same link most likely

### PR DESCRIPTION
**Pull request by slimbuck**
Fixes #220

This PR updates the model viewer as follows:
- update packages to latest versions
- adds a few additional scene stats
- fixes a rendering issue with projected skybox
- addresses a few issues with scene "popping" when loading new models
- QR code is larger and url is correct
- model-viewer version is updated to v3.2.0

New scene stats are shown below:
Screenshot 2023-06-12 at 14 09 33

### CODE REVIEW

File: src/ui/left-panel/index.tsx [hunk 1]
```diff
@@ -23,13 +23,27 @@ const openPanel = () => {
     }
 };
 
+const toSizeString = (value: number) => {
```
Comment (mvaligursky) [inline src/ui/left-panel/index.tsx:26]: could do with something like this https://gist.github.com/lanqy/5193417

File: src/viewer.ts [hunk 2]
```diff
@@ -827,33 +812,36 @@ class Viewer {
...
                     primitiveCount += mesh.primitive[0].count;
+                    meshVRAM += mesh.vertexBuffer.numBytes + (mesh.indexBuffer?.numBytes ?? 0);
                 });
             });
+
+            materialCount += asset.resource.materials.length;
+
+            textureCount += asset.resource.textures.length;
+            asset.resource.textures.forEach((texture: Asset) => {
+                textureVRAM += texture.resource.gpuSize;
```
Comment (mvaligursky) [inline src/viewer.ts:843]: Could textures be shared between assets here?

Comment (mvaligursky) [reply src/viewer.ts:843]: Yes but can they reference the same external images? Or would we load them multiple times in that case as well.

Comment (mvaligursky) [reply src/viewer.ts:843]: I guess you show stats per glb anyways, so that would make a difference.

File: src/ui/left-panel/index.tsx (hunk 1 above)
Comment (slimbuck) [reply src/ui/left-panel/index.tsx]: ok, switch to that version.

File: src/projective-skybox.ts [hunk 3]
```diff
@@ -43,9 +48,10 @@ void main(void) {
     vec3 view_dir = normalize(vViewDir);
 
     // intersect ray with world geometry
-    float t = 100.0;
-    if (view_dir.y < 0.0) intersectPlane(t, view_pos, view_dir, groundPlane);
-    intersectSphere(t, view_pos, view_dir, domeParams);
+    float t = 8000.0;
```
Comment (willeastcott) [inline src/projective-skybox.ts:51]: What's t? Why 8000?

**Review by willeastcott — APPROVED** [review]

File: src/projective-skybox.ts (hunk 3 above)
Comment (slimbuck) [reply src/projective-skybox.ts:51]: Maximum ray length

Comment (slimbuck) [reply src/projective-skybox.ts]: I've added a comment to explain as much.

**Pull request merged**